website/docs/reference/dbt_project.yml.md

```diff
@@ -6,6 +6,18 @@
 
 Every [dbt project](projects) needs a `dbt_project.yml` file — this is how dbt knows a directory is a dbt project. It also contains important information that tells dbt how to operate on your project.
 
+<VersionBlock lastVersion="1.4">
+
+By default, dbt will look in your current working directory for `dbt_project.yml`, but you can set a different directory using the `--profiles-dir` flag.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.5">
+
+By default, dbt will look in your current working directory for `dbt_project.yml`, but you can set a different directory using the `--profiles-dir` flag or the `DBT_PROJECT_DIR` environment variable.
+
+</VersionBlock>
```

Comment:
Add the fact that the parents of the current working directory will also be searched. In reality, it will continue to search until it either reaches the root or finds a `dbt_project.yml` file, but we don't need to go into that level of detail here.

```suggestion
<VersionBlock lastVersion="1.4">

By default, dbt will look for `dbt_project.yml` in your current working directory and its parents, but you can set a different directory using the `--profiles-dir` flag.

</VersionBlock>

<VersionBlock firstVersion="1.5">

By default, dbt will look for `dbt_project.yml` in your current working directory and its parents, but you can set a different directory using the `--profiles-dir` flag or the `DBT_PROJECT_DIR` environment variable.

</VersionBlock>
```